python/pyspark/sql/tests/connect/test_connect_function.py

```diff
@@ -2078,6 +2078,30 @@ def test_string_functions_multi_args(self):
         cdf = self.connect.sql(query)
         sdf = self.spark.sql(query)
 
+        # test is_valid_utf8
```

Comment:
you can just add the test at `test_functions.py`. The tests will be resused.

Reply:
moved tests to `test_functions.py`